ruby/private/bundle/create_bundle_build_file.rb

```diff
@@ -223,6 +241,13 @@ def generate!
                         .gsub('{bundle_deps}', gems.map { |g| ":#{g}" }.to_s)
                         .gsub('{exclude}', DEFAULT_EXCLUDES.to_s)
 
+    gems_by_group.each do |key, value|
+      template_out.puts GEM_GROUP
```

Comment:
You need to do `GEM_GROUP.dup` to bypass the frozen constant error. The way it's written right now, the code is attempting to modify the frozen constant.